Composer/packages/client/src/components/AppComponents/RightPanel.tsx

```diff
@@ -33,10 +37,20 @@ const content = css`
 
 const Content = forwardRef<HTMLDivElement>((props, ref) => <div css={content} {...props} ref={ref} />);
 
+const SHOW_TREE = ['design'];
```

Comment:
Nit:  My preference is to avoid the UPPER_CASE.  I camelCase them in my own projects because they are module level variables (that happen to be const). Variables that are scoped to a function are camelCase and often const.